apps/etterlatte-behandling/src/main/kotlin/sak/SakDao.kt

```diff
@@ -86,10 +86,19 @@ class SakDao(private val connectionAutoclosing: ConnectionAutoclosing) {
         }
     }
 
-    fun hentSaker(): List<Sak> {
+    fun hentSaker(
+        kjoering: String,
+        antall: Int,
+    ): List<Sak> {
         return connectionAutoclosing.hentConnection { connection ->
             with(connection) {
-                val statement = prepareStatement("SELECT id, sakType, fnr, enhet from sak")
+                val statement =
+                    prepareStatement(
+                        """SELECT id, sakType, fnr, enhet from sak s 
+                    where not exists(select 1 from kjoering k where k.sak_id=s.id and k.kjoering='$kjoering')
```

Comment:
Jeg leser dette riktig som at hvis vi rekjører regulering så vil ikke de som allerede har kjørt før (selv med feilet) bli kjørt på nytt?

Reply:
Ja, stemmer, tanken er å ikkje ta med dei som alt er køyrd i år, for å kunne dele det opp/senke risikoen ved å køyre regulering på nytt eller fleire gongar.

Feila har eg eigentleg ikkje tenkt spesielt på akkurat her, men vi bør jo finne noko også for dei. Det blir jo sikkert ein del i år med så stort volum.

Reply:
Eller, eg _hadde_ jo tenkt spesielt på feila. Ja, kanskje denne også skal ha ein where status != FEILA.